### ISSUE TITLE
`lerna init` should prompt for a packages location and create any directories specified

### ISSUE THREAD

**Issue by peggyrayzis** [lerna/lerna#704]
## Expected Behavior

Lerna should create an empty `packages/` directory when running `lerna init` as described in the README.

## Current Behavior

Lerna does not create an empty `packages/` directory when running `lerna init` as described in the README.

## Possible Solution

* When initializing an empty repo, Lerna should prompt for a "Packages location" (defaulting to `"./packages"`), and then create whatever is entered.
* When re-initializing a lerna monorepo, Lerna should _not_ prompt (if `this.repository.packagesLocation` is defined, don't overwrite it).

## Steps to Reproduce (for bugs)

![lerna test](https://cloud.githubusercontent.com/assets/18017067/24168784/dd3e9560-0e51-11e7-998d-848cc0c1eacc.gif)

<details><summary>lerna.json</summary><p>

```json
<!-- Please paste your `lerna.json` here -->
{
  "lerna": "2.0.0-beta.38",
  "packages": [
    "packages/*"
  ],
  "version": "0.0.0"
}

```
</p></details>

## Context

I had to create the packages directory myself!!! 😭  haha no big deal, just thought I would bring it to your attention!

## Your Environment

| Executable | Version |
| ---: | :--- |
| `lerna --version` | 2.0.0-beta.38 |
| `npm --version`  | 4.1.2 |
| `yarn --version` | 0.21.3 |
| `node --version` | 7.5.0 |

| OS | Version |
| --- | --- |
| NAME | VERSION |
| macOS El Capitan | 10.11.6 |

**Comment by evocateur** [comment]
@peggyrayzis This is a great bug report, thanks!

We've definitely let the README drift in this regard, as pointed out in #591 and potentially updated in #703 (it doesn't indicate that you should create the directory, it just omits any reference to it).

We can do better than this, methinks. Would a prompt for "Location of packages?" (when not already defined) that defaults to "./packages" be helpful?

cc @gigabo

**Comment by peggyrayzis** [comment]
> We can do better than this, methinks. Would a prompt for "Location of packages?" (when not already defined) that defaults to "./packages" be helpful?

That sounds reasonable to me! 😀

**Comment by peggyrayzis** [comment]
@evocateur 🙋 I appreciate all the hard work you all have put in toward this project, so I'll take care of this issue. We're loving lerna so far at [nteract](https://github.com/nteract/nteract) and Major League Soccer.

**Comment by evocateur** [comment]
@peggyrayzis Awesome!

**Comment by Krizzu** [comment]
Hey,

Is this issue still to grab?

**Comment by Siilwyn** [comment]
I'm working on it: https://github.com/Siilwyn/lerna/tree/cli-init-packages-dir

**Comment by Siilwyn** [comment]
Prompting for the packages location is complex since the actual packages location for the config is a glob it could be nested or spread. I've opened a PR #970 to just create the packages directory when `init` is ran in an empty directory and to not act when the packages glob is configured. So it does not resolve this issue completely...

**Comment by cezaraugusto** [comment]
closed by #970?

**Comment by Siilwyn** [comment]
Yes.
